lib/SparkPost/APIResource.php

```diff
@@ -174,29 +199,32 @@ private function buildBody($options) {
    * @param string $action HTTP method type
    * @param string $resourcePath (optional) string resource path of specific resource
    * @param array $options (optional) query string parameters
+   *
    * @return array Result set of action performed on resource
+   *
    * @throws APIResponseException
    */
-  private function callResource( $action, $resourcePath=null, $options=[] ) {
-    $action = strtoupper($action); // normalize
+  private function callResource($action, $resourcePath = null, $options = [])
+  {
+      $action = strtoupper($action); // normalize
 
     $url = $this->buildUrl($resourcePath, $options);
-    $body = $this->buildBody($options);
+      $body = $this->buildBody($options);
```

Comment:
I like most of what has been done here but I don't think that it handled the spaces to tabs thing very well.  These 3 lines end up having `$action` and `$body` both indented farther than `$url`